torch/_inductor/codegen/wrapper.py

```diff
@@ -638,7 +638,7 @@ def get_kernel_path(self, code):
         ext = "so"
         extra = cpp_compile_command("i", "o", vec_isa=picked_vec_isa)
         # \n is required to match with the CodeCache behavior
-        source_code = "\n" + code.getvalue()
+        source_code = "\n" + code.getrawvalue()
```

Comment:
Add a note on why `getrawvalue` is needed?

Reply:
Comments are added.